File: Java/refactoring1/src/main/java/za/co/entelect/refactoring1/domain/MoneyMarketAccount.java
package za.co.entelect.refactoring1.domain;

public class MoneyMarketAccount extends BankAccount {

    public MoneyMarketAccount(Long balanceInCents, double creditInterestsRate, double debitInterestRate) {
        super(balanceInCents, creditInterestsRate, debitInterestRate);
    }

    @Override
    public AccountType getAccountType() {
        return AccountType.MONEY_MARKET;
    }
}

File: C#/Refactoring/refactoring_exercise_1/za/co/entelect/refactoring1/domain/MoneyMarketAccount.cs
namespace refactoring_exercise_1.za.co.entelect.refactoring1.domain
{

    public class MoneyMarketAccount : BankAccount
    {

        public MoneyMarketAccount(long balanceInCents, double creditInterestsRate, double debitInterestRate, long accountFee) :
            base(balanceInCents, creditInterestsRate, debitInterestRate, accountFee)
        {

        }
        public override AccountType GetAccountType()
        {
            return AccountType.MoneyMarket;
        }
    }
}